stanford_events_importer.install

```diff
@@ -760,3 +760,11 @@ function stanford_events_importer_update_7304(&$sandbox) {
     field_cache_clear();
   }
 }
+
+/**
+ * Update variables to point to events-legacy.stanford.edu.
+ */
+function stanford_events_importer_update_7305(&$sandbox) {
```

Comment:
Anything else in the DB that would need to be updated? Any values stored in nodes?

Reply:
I don't think so, in terms of by default. For instance, I don't believe that nodes have self-referential links to `events.stanford.edu`, at least not from what I could tell from importing "Film" and SWS unpublished events.

If end-users have modified the content type, or entered links to `events.stanford.edu` in other fields, that's not something that I want to touch.

Reply:
Also need to update the `feeds_source` table.
![image](https://user-images.githubusercontent.com/7185045/149428601-5fcada26-bea4-4d22-b39d-fd40d8aeeb84.png)
Maybe update the `stanford_events_organization_list` table
![image](https://user-images.githubusercontent.com/7185045/149428654-82ce821e-530a-42c6-9b81-07edb053f994.png)

Reply:
OOTB we are also populating the `field_data_field_stanford_event_add_to_cal` field. But i'm not sure we want to mess with that.
![image](https://user-images.githubusercontent.com/7185045/149429131-99d7fefc-e844-4271-9669-7bdddcb6b28f.png)

Reply:
OK, I can take a look at those

Reply:
I *think* `feeds_source` gets populated by the variable that we're setting/updating. Here's a local install; I haven't touched that table:

```
drush sqlq --extra="-t" 'select * from feeds_source'
+-------------------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------+--------+----------------+------------+
| id                      | feed_nid | config                                                                                                                                                                      | source                                                                            | state  | fetcher_result | imported   |
+-------------------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------+--------+----------------+------------+
| stanford_event_importer |       77 | a:2:{s:16:"FeedsHTTPFetcher";a:1:{s:6:"source";s:63:"http://events-legacy.stanford.edu/xml/drupal/v2.php?&category=8";}s:19:"FeedsXPathParserXML";a:0:{}}                   | http://events-legacy.stanford.edu/xml/drupal/v2.php?&category=8                   | a:0:{} | b:0;           | 1641578159 |
| stanford_event_importer |       85 | a:2:{s:16:"FeedsHTTPFetcher";a:1:{s:6:"source";s:70:"http://events-legacy.stanford.edu/xml/drupal/v2.php?&organization=1679";}s:19:"FeedsXPathParserXML";a:0:{}}            | http://events-legacy.stanford.edu/xml/drupal/v2.php?&organization=1679            | a:0:{} | b:0;           | 1641578160 |
| stanford_event_importer |       86 | a:2:{s:16:"FeedsHTTPFetcher";a:1:{s:6:"source";s:81:"http://events-legacy.stanford.edu/xml/drupal/v2.php?&organization=1679&unlisted=1";}s:19:"FeedsXPathParserXML";a:0:{}} | http://events-legacy.stanford.edu/xml/drupal/v2.php?&organization=1679&unlisted=1 | a:0:{} | b:0;           | 1641578160 |
+-------------------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------+--------+----------------+------------+
```

Reply:
Rubber-ducking on this a little bit in between meetings. We could:

- Query the DB for known tables; unserialize data as necessary; save the modified values back to the DB. Moderate amount of work. _Pros_: clean; _cons_: run the risk of missing something, more work than it's worth?
- Use [sar](https://github.com/sherakama/sar) to search and replace in the DB. _Pros_: very thorough; _cons_: **very** thorough, also DB-intensive.
- Pull a list of `stanford_event_importer` nodes, run a `node_save()` on each, and then run cron. _Pros_: probably the easiest to implement, and the lightest touch; _cons_: potential for missing stuff, may also be dependent on the next update from `events-legacy.stanford.edu`.

I am going to work on that last approach and see where it gets me.

Reply:
Note to self: do **not** import "Arts" events, unless you want 1164 nodes created.

Reply:
- If it were me, i'd just do the DB queries to do the serializing and updates. 
- I don't think `sar` would be good here.
- Loading nodes that are `stanford_event_importer` might miss node types that have been created for various reasons. So it could miss some importers.

Reply:
>     * I don't think `sar` would be good here.

Yeah, agreed. Last resort, and honestly I would want to put together a much longer list of pros and cons before doing so.

>     * Loading nodes that are `stanford_event_importer` might miss node types that have been created for various reasons. So it could miss some importers.

"Tell me more" - I'm not quite picking up what you're putting down here.

Reply:
Oh, other content types that site admins may have created to pull from `events.stanford.edu`?

If that's the case, that's on them. I don't feel responsible for updating those, and TBH I would rather not. Same reason I shy away from `drush sar`...